woocommerce/woocommerce
# [Product Blocks editor]: add tooltip help to Organization/Product catalog and Attributes

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds descriptions (tooltip help) to the _Product catalog_ and `Attributes` section of the _Organization_ page.

Closes https://github.com/woocommerce/woocommerce/issues/39636



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the Product Editor
2. Click on the _Organization_ tab
3. Confirm you see the `(?)` button next to the both Product catalog and Attributes sections
4. Confirm you see the help tooltip by clicking on the buttons:

<img width="779" alt="Screenshot 2023-11-22 at 15 10 36" src="https://github.com/woocommerce/woocommerce/assets/77539/aa56dbd7-c874-4d04-927f-4783a588b28d">

<img width="742" alt="Screenshot 2023-11-22 at 15 10 19" src="https://github.com/woocommerce/woocommerce/assets/77539/0c099c2a-78b8-46ba-b52f-dbfea40fb2ed">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
